code/modules/reagents/reagents/reagents_ethanol.dm

```diff
@@ -853,6 +853,35 @@
 	glass_icon_state = "aleglass"
 	glass_desc = "A cold pint of delicious ale."
 
+/datum/reagent/ethanol/drink/cider
+	name = "Cider"
+	id = CIDER
+	description = "Alcoholic, fermented apples."
+	nutriment_factor = 2 * REAGENTS_METABOLISM
+	color = "#ffee88"
+	glass_icon_state = "ciderglass"
+	glass_desc = "The hard kind. Alcoholic."
+
+/datum/reagent/ethanol/drink/scrumpy
+	name = "Scrumpy"
+	id = SCRUMPY
+	description = "Alcoholic, fermented lemons."
```

Comment:
<img width="309" height="500" alt="image" src="https://github.com/user-attachments/assets/21c83cb5-f8ce-43b5-8b2a-1bf192b9274b" />

Reply:
oh wew
might as well remove it then, bit out of scope anyways